app.js

```diff
@@ -98,6 +128,11 @@ function onClick(event){
     shownProducts[`${id[5]}`].imgTimesClicked++;
     generateAndRenderImage();
   }
+  if(localStorage.getItem('productData')){
+    OddDuckProduct.prototype.loadPriorData();
```

Comment:
Methods need to be called on specific objects, when you call it here your code has no idea what "this" is in the code block.

Additionally, this shouldn't be in an event listener. You only need to get your data once, and you only need to save it once.

Reply:
Ok. David and Adam were under the impression that data needed to be saved on every click, this is why I was calling my save functions in the click handler. Thank you!